sound/soc/sof/ipc4-topology.c

```diff
@@ -1726,9 +1726,12 @@ sof_ipc4_prepare_copier_module(struct snd_sof_widget *swidget,
 			SOF_IPC4_MIN_DMA_BUFFER_SIZE * copier_data->base_config.ibs;
 		break;
 	case snd_soc_dapm_aif_in:
-			copier_data->gtw_cfg.dma_buffer_size =
-				max((u32)SOF_IPC4_MIN_DMA_BUFFER_SIZE, deep_buffer_dma_ms) *
-					copier_data->base_config.ibs;
+		copier_data->gtw_cfg.dma_buffer_size =
+			max((u32)SOF_IPC4_MIN_DMA_BUFFER_SIZE, deep_buffer_dma_ms) *
+				copier_data->base_config.ibs;
+		dev_dbg(sdev->dev, "copier %s, deep_buffer_dma_ms %u, dma_buffer_size %u",
```

Comment:
if we are not using deep buffer then printing
```
copier host-copier.1.playback, deep_buffer_dma_ms 2, dma_buffer_size 384
```
does not make much sense, imho
If we just drop the deep from the print:
```
copier host-copier.1.playback, dma_buffer_ms 2, dma_buffer_size 384
```
or, what about:
```
copier host-copier.1.playback, dma buffer: 2 ms (384 bytes)
...
copier host-copier.1.playback, dma buffer (using Deep Buffer): 10 ms (1920 bytes)
```
```c
		dev_dbg(sdev->dev, "copier %s, dma buffer%s: %u ms (%u bytes)",
			swidget->widget->name,
			deep_buffer_dma_ms ? " (using Deep Buffer)" : "",
			max((u32)SOF_IPC4_MIN_DMA_BUFFER_SIZE, deep_buffer_dma_ms),
			copier_data->gtw_cfg.dma_buffer_size);
```

Reply:
Ack, prefer the last version for clarity, will update.